pkg/server/gitproviders/gitprovider.go

```diff
@@ -21,15 +21,16 @@ func (s *GitProviderService) GetGitProviderForUrl(repoUrl string) (gitprovider.G
 	}
 
 	for _, p := range gitProviders {
-		if p.Id == "aws-codecommit" && strings.Contains(repoUrl, "git-codecommit") {
+		pId := strings.Split(p.Id, "_+_")[0]
```

Comment:
You're completely right. I believe we can remove `TokenScope` and `TokenScopeType`. Since we're going to do the matching ourselves, I believe there's no reason for the user to choose the type. We can treat all tokens as "global".

Reply:
https://github.com/daytonaio/daytona/pull/1032#discussion_r1754413691

This comment becomes obsolete then as well.

P.S. Please don't mark conversations as resolved yourself. It's easier for us to track.